docs/PacketFence_OpenWrt-Hostapd_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,152 @@
+OpenWrt BarrierBreaker 14.07 with hostapd Quick Integration Guide
+=================================================================
+:encoding: UTF-8
+:lang: en
+:doctype: book
+
+include::includes/global-attributes.asciidoc[]
+
+About this Guide
+----------------
+This guide has been created in order to help sales engineers, product managers, or network specialists demonstrate the PacketFence capabilities on-site with an existing or potential customer.  It can also provide guidelines to setup a proof of concept for a potential PacketFence deployment using OpenWrt BarrierBreaker 14.07 with Hostapd.
+
+Introduction
+------------
+This guide will provide an example for the configuration of an open SSID (not encrypted) and a secured SSID (802.1x). You will to install wpad and hostapd. These two SSIDs will do RADIUS authentication against PacketFence.
+
+Assumptions
+-----------
+* You have a configured PacketFence environment with working test equipment
+* The management IP of PacketFence will be 192.168.1.10 and has s3cr3t as its RADIUS shared secret
+* You have an acces point with OpenWrt BarrierBreaker 14.07 installed
+
+
+Quick installation
+------------------
+Step 1: Packages installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can install the packages from the web interface of OpenWrt.
+
+Go to *System -> Software*
+
+First update the repos by clicking the button Update lists.
+
+Then you will have to install the packages of Hostapd and wpad.
+
+Go to the tab 'Available packages' and then search for the package hostapd into the 'Filter:' field.
+
+Click Install the hostapd package, the actual version is 2014-06-03.1-1.
+
+Do the same process for the wpad package version 2014-06-03.1-1.
+
+Step 2: Dynamic VLAN Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Create the file : /etc/config/hostapd.vlan
+----
+*           wlan0.#
+----
+
+Step 3: Hostapd configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You will need to modify the hostapd script that comes with the package that we previously installed.
+
+Connect using SSH to the AP and run these commands:
+----
+cd /lib/netifd/
+mv hostapd.sh hostapd.sh.old
+wget https://github.com/inverse-inc/packetfence/tree/devel/addons/hostapd/hostapd-14.07.sh
```

Comment:
not an http or ftp url:
I tried differents urls and some with https none are working, the dns resolution is only made for http tho